akka-http/src/main/scala/akka/http/scaladsl/server/RejectionHandler.scala

```diff
@@ -165,7 +165,11 @@ object RejectionHandler {
       }
       .handle {
         case TooManyRangesRejection(_) ⇒
-          complete((RequestedRangeNotSatisfiable, "Request contains too many ranges."))
+          complete((RequestedRangeNotSatisfiable, "Request contains too many ranges"))
+      }
+      .handle {
+        case CircuitBreakerOpenRejection(_) ⇒
+          complete((EnhanceYourCalm, "Circuit Breaker is open; calls are failing fast"))
```

Comment:
I'd just do `complete(InternalServerError)` as rate limiting is a special usage of the CB. `EnhanceYourCalm `is rather non-standard. Would `TooManyRequests` be better suited for the rate limiting case?